### DISCUSSION TITLE
Trying to integrate with tig

### DISCUSSION THREAD

**Discussion by ghost** [dandavison/delta#689] [Ideas]
Hi. I've been trying to integrate with tig.

https://github.com/jonas/tig/issues/542#issuecomment-898663408

![ss 0003-08-14 at 3 54 11](https://user-images.githubusercontent.com/41639488/129411078-b86543de-a5d7-47a7-86ce-c933879fb918.png)

![ss 0003-08-14 at 3 58 35](https://user-images.githubusercontent.com/41639488/129411090-4765c710-554c-4d0e-bd4d-d2ee96f554e5.png)

I'm facing the color problem. delta color is user-definable, so there are (256^3)^2(fg and bg) possibilities, but defining these combinations in the initialization of tig is impractical.

I think I may have to restrict delta inside tig with only one theme, or wonder if there is a better way ...

Any ideas are appreciated.

**Comment by ghost** [comment]
true color never may work. I'm going to try.

**Comment by ghost** [comment]
we need to fix build option for this part.

https://github.com/mirror/ncurses/blob/master/ncurses/base/lib_color.c#L382-L391

**Comment by ghost** [comment]
It's almost there.

![ss 0003-08-17 at 16 53 28](https://user-images.githubusercontent.com/41639488/129687856-84d59825-c14e-4030-8484-d798d3a5eb45.png)

**Comment by ghost** [comment]
maybe finished
![ss 0003-08-17 at 19 11 58](https://user-images.githubusercontent.com/41639488/129710912-f5f1fd99-1b4f-4b51-bdca-9301f0fd7e65.png)


https://github.com/jonas/tig/pull/1140

**Comment by KonradStanski** [comment]
Any update on this?
